pyfstat/grid_based_searches.py

```diff
@@ -1161,6 +1234,248 @@ def _get_savetxt_fmt_dict(self):
         return fmt_dict
 
 
+class SearchOverGridFile(TransientGridSearch):
+    """A search over a grid defined in an input text file.
+
+    Such a file can be generated for example with
+    lalapps_ComputeFstatistic_v2 using the --outputGrid option.
+
+    WARNING: The header file of the grid file is NOT yet parsed
+    for consistency of CFSv2 options (such as reference time)
+    with the PyFstat options,
+    so please ensure yourself that these match!
```

Comment:
This would be a nice future enhancement.

(Leaving comment here so I can then create an issue from it.)